ads/google/utils.js

```diff
@@ -43,22 +55,31 @@ export const ADSENSE_MCRSPV_TAG = 'mcrspv';
  * @param {boolean} multiSizeValidation A flag that if set to true will enforce
  *   the rule that ensures multi-size dimensions are no less than 2/3rds of
  *   their primary dimension's counterpart.
- * @param {boolean=} isFluid Indicates whether this ad slot is Fluid-enabled.
+ * @param {boolean=} isFluidPrimary Indicates whether the ad slot's primary
+ *   size is fluid.
  * @return {?Array<!Array<number>>} An array of dimensions.
  */
 export function getMultiSizeDimensions(
   multiSizeDataStr,
   primaryWidth,
   primaryHeight,
   multiSizeValidation,
-  isFluid = false) {
+  isFluidPrimary = false) {
 
   const dimensions = [];
   const arrayOfSizeStrs = multiSizeDataStr.split(',');
 
   for (let i = 0; i < arrayOfSizeStrs.length; i++) {
 
     const sizeStr = arrayOfSizeStrs[i];
+    if (sizeStr.toLowerCase() == 'fluid') {
+      if (!isFluidPrimary) {
+        // If the primary size is fluid, then the dummy 320x50 size will
```

Comment:
nit: probably better not to repeat 320x50 here?

Reply:
Done.